src/gpu/multiexp.rs

```diff
@@ -111,7 +113,11 @@ where
 {
     pub fn create(d: Device, priority: bool) -> GPUResult<SingleMultiexpKernel<E>> {
         let src = sources::kernel::<E>();
-        let pq = ProQue::builder().device(d).src(src).dims(1).build()?;
+
+        let platform = match d.info(ocl::enums::DeviceInfo::Platform)? {
```

Comment:
Hmm, why would we need to set the platform when the device we are going to use is already given to the create function?